aten/src/ATen/native/BatchLinearAlgebraKernel.cpp

```diff
@@ -1018,6 +1018,7 @@ static void apply_svd(const Tensor& A,
 void svd_kernel(const Tensor& A,
                 const bool full_matrices,
                 const bool compute_uv,
+                c10::optional<c10::string_view> driver,
```

Comment:
Add a `TORCH_INTERNAL_ASSERT` that this driver does not have a value, just in case.